app/presenters/tagging_spreadsheet_presenter.rb

```diff
@@ -18,4 +18,8 @@ def data_attributes
       'original-title': error_message
     }
   end
+
+  def errored?
+    state == 'errored'
+  end
 end
```

Comment:
@Davidslv this class also has a `data_attributes` to remove: https://github.com/alphagov/content-tagger/blob/12195a1276575678c8e7de44df17612886c74926/app/presenters/tagging_spreadsheet_presenter.rb#L13-L20